components/next_steps_view/next_steps_tips.tsx

```diff
@@ -84,12 +87,22 @@ const getDownloadButtonString = () => {
 
     return (
         <FormattedMessage
-            id='next_steps_view.tips.downloadForDefault'
-            defaultMessage='Download Mattermost'
+            id='next_steps_view.tips.getForDefault'
+            defaultMessage='Get Mattermost'
         />
     );
 };
 
+const openAdminConsole = (isAdmin: boolean) => {
+    trackEvent(getAnalyticsCategory(isAdmin), 'click_admin_console');
+    browserHistory.push('/admin_console/');
+};
+
+const openIncidentsPlugin = (isAdmin: boolean, team: Team) => {
+    trackEvent(getAnalyticsCategory(isAdmin), 'click_open_incidents');
+    browserHistory.push(`/${team.name}/com.mattermost.plugin-incident-management/playbooks/`);
```

Comment:
@lieut-data here we are linking to the playbooks backstage of IC. Just wanted to get your perspective if there was a better way to link there. Worried about plugin id changes and this link failing.

Reply:
I think this is the best we can do right now -- thanks for the heads up!